gradle.properties

```diff
@@ -1,7 +1,6 @@
 kotlin.code.style=official
 kotlin.incremental.js=true
 kotlin.incremental.multiplatform=true
-kotlin.parallel.tasks.in.project=true
```

Comment:
question

is this going to slow things down?  this was removed to support benchmarks?

Reply:
No this was removed because of a warning being generated, this property is taken by default from the `org.gradle.parallel` (`--parallel`) flag now by default and does not need to be specified separately